<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddModel</class>
 <widget class="QDialog" name="AddModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление модели авто</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 54, 54)</string>
  </property>
  <widget class="QPushButton" name="add_model_but">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167)</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Модель</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="model">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(243,243,243)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
